src/main/steps/applicant1/check-your-answers/content.ts

```diff
@@ -547,6 +557,7 @@ const cy: typeof en = ({
     },
     dividingAssets: {
       line1: 'Do you want to apply for a financial order?',
+      line2: 'Who is the financial for?',
```

Comment:
ignore this if it's right, but looks like it's missing an 'order' and should be 'Who is the financial order for?'

Reply:
yep my bad, i wrote the question wrong and copied it from there.